docker/build-tools/Dockerfile

```diff
@@ -359,9 +359,10 @@ ENV GO111MODULE=on
 ENV GOPROXY=https://proxy.golang.org
 ENV GOSUMDB=sum.golang.org
 ENV GOROOT=/usr/local/go
-ENV GOPATH=/home/go
-ENV GOCACHE=/home/gocache
-ENV PATH=/usr/local/go/bin:/home/go/bin:/usr/local/google-cloud-sdk/bin:$PATH
+ENV GOPATH=/go
+ENV GOCACHE=/gocache
+ENV GOBIN=/tmp/out
+ENV PATH=/usr/local/go/bin:/tmp/out:/usr/local/google-cloud-sdk/bin:$PATH
```

Comment:
if /tmp is a bindmount, this is dangerous, as anyone could put a binary in /tmp/out in a shared environment and override the toolchain...